# -*- coding: utf-8 -*-

from PyQt5 import QtCore, QtGui, QtWidgets

class Ui_QSubWindow_default(object):
    def setupUi(self, QSubWindow_default):
        QSubWindow_default.setObjectName("QSubWindow_default")
        QSubWindow_default.resize(590, 345)
        self.horizontalLayout_15 = QtWidgets.QHBoxLayout(QSubWindow_default)
        self.horizontalLayout_15.setObjectName("horizontalLayout_15")
        self.verticalLayout_7 = QtWidgets.QVBoxLayout()
        self.verticalLayout_7.setObjectName("verticalLayout_7")
        self.horizontalWidget_8 = QtWidgets.QWidget(QSubWindow_default)
        self.horizontalWidget_8.setObjectName("horizontalWidget_8")
        self.horizontalLayout_8 = QtWidgets.QHBoxLayout(self.horizontalWidget_8)
        self.horizontalLayout_8.setObjectName("horizontalLayout_8")
        spacerItem = QtWidgets.QSpacerItem(40, 20, QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Minimum)
        self.horizontalLayout_8.addItem(spacerItem)
        self.checkBox_3 = QtWidgets.QCheckBox(self.horizontalWidget_8)
        self.checkBox_3.setObjectName("checkBox_3")
        self.horizontalLayout_8.addWidget(self.checkBox_3)
        spacerItem1 = QtWidgets.QSpacerItem(40, 20, QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Minimum)
        self.horizontalLayout_8.addItem(spacerItem1)
        self.checkBox_4 = QtWidgets.QCheckBox(self.horizontalWidget_8)
        self.checkBox_4.setObjectName("checkBox_4")
        self.horizontalLayout_8.addWidget(self.checkBox_4)
        spacerItem2 = QtWidgets.QSpacerItem(40, 20, QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Minimum)
        self.horizontalLayout_8.addItem(spacerItem2)
        self.dateEdit_3 = QtWidgets.QDateEdit(self.horizontalWidget_8)
        self.dateEdit_3.setObjectName("dateEdit_3")
        self.horizontalLayout_8.addWidget(self.dateEdit_3)
        spacerItem3 = QtWidgets.QSpacerItem(40, 20, QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Minimum)
        self.horizontalLayout_8.addItem(spacerItem3)
        self.label_5 = QtWidgets.QLabel(self.horizontalWidget_8)
        self.label_5.setObjectName("label_5")
        self.horizontalLayout_8.addWidget(self.label_5)
        spacerItem4 = QtWidgets.QSpacerItem(40, 20, QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Minimum)
        self.horizontalLayout_8.addItem(spacerItem4)
        self.dateEdit_4 = QtWidgets.QDateEdit(self.horizontalWidget_8)
        self.dateEdit_4.setObjectName("dateEdit_4")
        self.horizontalLayout_8.addWidget(self.dateEdit_4)
        spacerItem5 = QtWidgets.QSpacerItem(40, 20, QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Minimum)
        self.horizontalLayout_8.addItem(spacerItem5)
        self.label_6 = QtWidgets.QLabel(self.horizontalWidget_8)
        self.label_6.setObjectName("label_6")
        self.horizontalLayout_8.addWidget(self.label_6)
        spacerItem6 = QtWidgets.QSpacerItem(40, 20, QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Minimum)
        self.horizontalLayout_8.addItem(spacerItem6)
        self.verticalLayout_7.addWidget(self.horizontalWidget_8)
        self.verticalWidget_5 = QtWidgets.QWidget(QSubWindow_default)
        self.verticalWidget_5.setObjectName("verticalWidget_5")
        self.verticalLayout_4 = QtWidgets.QVBoxLayout(self.verticalWidget_5)
        self.verticalLayout_4.setObjectName("verticalLayout_4")
        self.horizontalWidget_9 = QtWidgets.QWidget(self.verticalWidget_5)
        self.horizontalWidget_9.setObjectName("horizontalWidget_9")
        self.horizontalLayout_9 = QtWidgets.QHBoxLayout(self.horizontalWidget_9)
        self.horizontalLayout_9.setObjectName("horizontalLayout_9")
        spacerItem7 = QtWidgets.QSpacerItem(40, 20, QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Minimum)
        self.horizontalLayout_9.addItem(spacerItem7)
        self.lineEdit_3 = QtWidgets.QLineEdit(self.horizontalWidget_9)
        self.lineEdit_3.setObjectName("lineEdit_3")
        self.horizontalLayout_9.addWidget(self.lineEdit_3)
        self.pushButton_3 = QtWidgets.QPushButton(self.horizontalWidget_9)
        self.pushButton_3.setObjectName("pushButton_3")
        self.horizontalLayout_9.addWidget(self.pushButton_3)
        spacerItem8 = QtWidgets.QSpacerItem(40, 20, QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Minimum)
        self.horizontalLayout_9.addItem(spacerItem8)
        self.verticalLayout_4.addWidget(self.horizontalWidget_9)
        self.horizontalWidget_10 = QtWidgets.QWidget(self.verticalWidget_5)
        self.horizontalWidget_10.setObjectName("horizontalWidget_10")
        self.horizontalLayout_10 = QtWidgets.QHBoxLayout(self.horizontalWidget_10)
        self.horizontalLayout_10.setObjectName("horizontalLayout_10")
        spacerItem9 = QtWidgets.QSpacerItem(40, 20, QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Minimum)
        self.horizontalLayout_10.addItem(spacerItem9)
        self.lineEdit_4 = QtWidgets.QLineEdit(self.horizontalWidget_10)
        self.lineEdit_4.setObjectName("lineEdit_4")
        self.horizontalLayout_10.addWidget(self.lineEdit_4)
        self.pushButton_4 = QtWidgets.QPushButton(self.horizontalWidget_10)
        self.pushButton_4.setObjectName("pushButton_4")
        self.horizontalLayout_10.addWidget(self.pushButton_4)
        spacerItem10 = QtWidgets.QSpacerItem(40, 20, QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Minimum)
        self.horizontalLayout_10.addItem(spacerItem10)
        self.verticalLayout_4.addWidget(self.horizontalWidget_10)
        self.verticalLayout_7.addWidget(self.verticalWidget_5)
        self.horizontalWidget_11 = QtWidgets.QWidget(QSubWindow_default)
        self.horizontalWidget_11.setObjectName("horizontalWidget_11")
        self.horizontalLayout_11 = QtWidgets.QHBoxLayout(self.horizontalWidget_11)
        self.horizontalLayout_11.setObjectName("horizontalLayout_11")
        spacerItem11 = QtWidgets.QSpacerItem(40, 20, QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Minimum)
        self.horizontalLayout_11.addItem(spacerItem11)
        self.verticalWidget = QtWidgets.QWidget(self.horizontalWidget_11)
        self.verticalWidget.setObjectName("verticalWidget")
        self.verticalLayout_5 = QtWidgets.QVBoxLayout(self.verticalWidget)
        self.verticalLayout_5.setObjectName("verticalLayout_5")
        self.radioButton_5 = QtWidgets.QRadioButton(self.verticalWidget)
        self.radioButton_5.setObjectName("radioButton_5")
        self.verticalLayout_5.addWidget(self.radioButton_5)
        self.radioButton_6 = QtWidgets.QRadioButton(self.verticalWidget)
        self.radioButton_6.setObjectName("radioButton_6")
        self.verticalLayout_5.addWidget(self.radioButton_6)
        self.radioButton_7 = QtWidgets.QRadioButton(self.verticalWidget)
        self.radioButton_7.setObjectName("radioButton_7")
        self.verticalLayout_5.addWidget(self.radioButton_7)
        self.radioButton_8 = QtWidgets.QRadioButton(self.verticalWidget)
        self.radioButton_8.setObjectName("radioButton_8")
        self.verticalLayout_5.addWidget(self.radioButton_8)
        self.horizontalLayout_11.addWidget(self.verticalWidget)
        self.verticalWidget_6 = QtWidgets.QWidget(self.horizontalWidget_11)
        self.verticalWidget_6.setObjectName("verticalWidget_6")
        self.verticalLayout_9 = QtWidgets.QVBoxLayout(self.verticalWidget_6)
        self.verticalLayout_9.setObjectName("verticalLayout_9")
        self.horizontalWidget_12 = QtWidgets.QWidget(self.verticalWidget_6)
        self.horizontalWidget_12.setObjectName("horizontalWidget_12")
        self.horizontalLayout_12 = QtWidgets.QHBoxLayout(self.horizontalWidget_12)
        self.horizontalLayout_12.setObjectName("horizontalLayout_12")
        self.spinBox_2 = QtWidgets.QSpinBox(self.horizontalWidget_12)
        self.spinBox_2.setObjectName("spinBox_2")
        self.horizontalLayout_12.addWidget(self.spinBox_2)
        self.label_7 = QtWidgets.QLabel(self.horizontalWidget_12)
        self.label_7.setObjectName("label_7")
        self.horizontalLayout_12.addWidget(self.label_7)
        self.verticalLayout_9.addWidget(self.horizontalWidget_12)
        self.horizontalWidget_13 = QtWidgets.QWidget(self.verticalWidget_6)
        self.horizontalWidget_13.setObjectName("horizontalWidget_13")
        self.horizontalLayout_13 = QtWidgets.QHBoxLayout(self.horizontalWidget_13)
        self.horizontalLayout_13.setObjectName("horizontalLayout_13")
        self.doubleSpinBox_2 = QtWidgets.QDoubleSpinBox(self.horizontalWidget_13)
        self.doubleSpinBox_2.setObjectName("doubleSpinBox_2")
        self.horizontalLayout_13.addWidget(self.doubleSpinBox_2)
        self.label_8 = QtWidgets.QLabel(self.horizontalWidget_13)
        self.label_8.setObjectName("label_8")
        self.horizontalLayout_13.addWidget(self.label_8)
        self.verticalLayout_9.addWidget(self.horizontalWidget_13)
        self.horizontalLayout_11.addWidget(self.verticalWidget_6)
        spacerItem12 = QtWidgets.QSpacerItem(40, 20, QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Minimum)
        self.horizontalLayout_11.addItem(spacerItem12)
        self.verticalLayout_7.addWidget(self.horizontalWidget_11)
        self.horizontalWidget_14 = QtWidgets.QWidget(QSubWindow_default)
        self.horizontalWidget_14.setObjectName("horizontalWidget_14")
        self.horizontalLayout_14 = QtWidgets.QHBoxLayout(self.horizontalWidget_14)
        self.horizontalLayout_14.setObjectName("horizontalLayout_14")
        spacerItem13 = QtWidgets.QSpacerItem(40, 20, QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Minimum)
        self.horizontalLayout_14.addItem(spacerItem13)
        self.progressBar_2 = QtWidgets.QProgressBar(self.horizontalWidget_14)
        self.progressBar_2.setProperty("value", 24)
        self.progressBar_2.setObjectName("progressBar_2")
        self.horizontalLayout_14.addWidget(self.progressBar_2)
        self.buttonBox_2 = QtWidgets.QDialogButtonBox(self.horizontalWidget_14)
        self.buttonBox_2.setOrientation(QtCore.Qt.Horizontal)
        self.buttonBox_2.setStandardButtons(QtWidgets.QDialogButtonBox.Cancel|QtWidgets.QDialogButtonBox.Ok)
        self.buttonBox_2.setObjectName("buttonBox_2")
        self.horizontalLayout_14.addWidget(self.buttonBox_2)
        spacerItem14 = QtWidgets.QSpacerItem(40, 20, QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Minimum)
        self.horizontalLayout_14.addItem(spacerItem14)
        self.verticalLayout_7.addWidget(self.horizontalWidget_14)
        self.horizontalLayout_15.addLayout(self.verticalLayout_7)

        self.retranslateUi(QSubWindow_default)
        QtCore.QMetaObject.connectSlotsByName(QSubWindow_default)

    def retranslateUi(self, QSubWindow_default):
        _translate = QtCore.QCoreApplication.translate
        QSubWindow_default.setWindowTitle(_translate("QSubWindow_default", "Dialog"))
        self.checkBox_3.setToolTip(_translate("QSubWindow_default", "<html><head/><body><p>Selects all Images in the Import Folder</p></body></html>"))
        self.checkBox_3.setText(_translate("QSubWindow_default", "Select all Images"))
        self.checkBox_4.setToolTip(_translate("QSubWindow_default", "<html><head/><body><p>Check to Select Select Start and End Date</p></body></html>"))
        self.checkBox_4.setText(_translate("QSubWindow_default", "Select Period of time"))
        self.dateEdit_3.setToolTip(_translate("QSubWindow_default", "<html><head/><body><p>Select Start Date</p></body></html>"))
        self.label_5.setText(_translate("QSubWindow_default", "Start Date"))
        self.dateEdit_4.setToolTip(_translate("QSubWindow_default", "<html><head/><body><p>Select End Date</p></body></html>"))
        self.label_6.setText(_translate("QSubWindow_default", "End Date"))
        self.pushButton_3.setToolTip(_translate("QSubWindow_default", "<html><head/><body><p>Browse to Import Folder</p></body></html>"))
        self.pushButton_3.setText(_translate("QSubWindow_default", "Browse"))
        self.pushButton_4.setToolTip(_translate("QSubWindow_default", "<html><head/><body><p>Browse to Output Folder</p></body></html>"))
        self.pushButton_4.setText(_translate("QSubWindow_default", "Browse"))
        self.radioButton_5.setToolTip(_translate("QSubWindow_default", "<html><head/><body><p>Check for Setting 1</p></body></html>"))
        self.radioButton_5.setText(_translate("QSubWindow_default", "Setting 1"))
        self.radioButton_6.setToolTip(_translate("QSubWindow_default", "<html><head/><body><p>Check for Setting 2</p></body></html>"))
        self.radioButton_6.setText(_translate("QSubWindow_default", "Setting 2"))
        self.radioButton_7.setToolTip(_translate("QSubWindow_default", "<html><head/><body><p>Check for Setting 3</p></body></html>"))
        self.radioButton_7.setText(_translate("QSubWindow_default", "Setting 3"))
        self.radioButton_8.setToolTip(_translate("QSubWindow_default", "<html><head/><body><p>Check for Setting 4</p></body></html>"))
        self.radioButton_8.setText(_translate("QSubWindow_default", "Setting 4"))
        self.label_7.setToolTip(_translate("QSubWindow_default", "<html><head/><body><p>Set Factor 1; e.g. Multilooking Factor</p></body></html>"))
        self.label_7.setText(_translate("QSubWindow_default", "Set Factor 1"))
        self.label_8.setToolTip(_translate("QSubWindow_default", "<html><head/><body><p>Set Factor 2</p></body></html>"))
        self.label_8.setText(_translate("QSubWindow_default", "Set Factor 2"))
        self.buttonBox_2.setToolTip(_translate("QSubWindow_default", "<html><head/><body><p>Press Ok to Start Processing</p><p>Press Cancel to Quit Dialog</p></body></html>"))

    def create_Ui_QSubWindow_default(self):
        QSubWindow_default = QtWidgets.QDialog()
        ui = Ui_QSubWindow_default()
        ui.setupUi(QSubWindow_default)
        QSubWindow_default.exec_()



if __name__ == "__main__":
    import sys
    app = QtWidgets.QApplication(sys.argv)
    QSubWindow_default = QtWidgets.QDialog()
    ui = Ui_QSubWindow_default()
    ui.setupUi(QSubWindow_default)
    QSubWindow_default.show()
    sys.exit(app.exec_())

